location-proof-protocol.md

```diff
@@ -0,0 +1,77 @@
+#########################
+
+## For review only
+
+#########################
+
+# 📍 Location Proof Protocol
+
+The Location Proof Protocol aims to address the challenge of proving location on the decentralized web in a flexible, yet standardized way.
```

Comment:
```suggestion
The Location Proof Protocol aims to address the challenge of proving location on the web, using the decentralized web in a flexible, yet standardized way.
```
..along those lines, to laminate that the problem is not only on web3, but it's a general problem on the internet, still, maybe just distracts focus, whatever is best.